testutil-server/src/test/java/io/spine/testing/server/expected/CommandHandlerExpectedShould.java

```diff
@@ -49,7 +49,7 @@
     void trackRejection() {
```

Comment:
We don't have `...Should`-named classes anymore. They all are called `...Test`. Please review the codebase you have recently created and update.

Reply:
Should I leave `shoulds` in the cases where other wise I'd get `...TestTest`?